core/tracing/CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ All notable changes to the tracing interface will be documented in this file.
 - `GasChangeReason` has been extended with the following reasons which will be enabled only post-Verkle. There shouldn't be any gas changes with those reasons prior to the fork.
   - `GasChangeWitnessContractCollisionCheck` flags the event of adding to the witness when checking for contract address collision.
 
+## [v1.14.12]
+
+This release contains a change in behavior for `OnCodeChange` hook.
+
+### `OnCodeChange` change
+
+The `OnCodeChange` is now called when `StateDB#SelfDestruct` or `StateDB#SelfDestruct6780` is invoked, which happens when a contract self destruct itself. Previously, no code change was emitted on such occasions.
```

Comment:
It's not accurate.

OnCodeChange is invoked only when StateDB#SelfDestruct is called, or when StateDB#SelfDestruct6780 is called and the contract is actually destroyed.